src/config.ts

```diff
@@ -54,6 +54,26 @@ export interface ValidationSchemaPluginConfig extends TypeScriptPluginConfig {
    * ```
    */
   importFrom?: string
+  /**
+   * @description If defined, will use named imports from the specified module (defined in `importFrom`)
+   * rather than individual imports for each type.
+   *
+   * @exampleMarkdown
+   * ```yml
+   * generates:
+   *   path/to/types.ts:
+   *     plugins:
+   *       - typescript
+   *   path/to/schemas.ts:
+   *     plugins:
+   *       - graphql-codegen-validation-schema
+   *     config:
+   *       schema: yup
+   *       importFrom: ./path/to/types
+   *       namespacedImportName: types
+   * ```
+   */
+  schemaNamespacedImportName?: string
```

Comment:
@jocisneros Could you add to also README?

Reply:
just added 🫡